BYTEROVER.md

```
@@ -0,0 +1,121 @@
+ # Byterover Handbook
+
+ *Generated: 2025-09-06*
+
+ ## Layer 1: System Overview
+
+ **Purpose**: Anonymous, mobile-first social app for sharing "secrets" (text or video confessions) with privacy protections (face blur, voice change), discovery via trending hashtags/secrets, and lightweight social interactions (likes, replies, saves). Auth, data, and storage are powered by Supabase.
+
+ **Tech Stack**: Expo SDK 53, React Native 0.79, React 19, TypeScript, React Navigation (native + tabs), Zustand (persisted with AsyncStorage), @shopify/flash-list, Reanimated 3, Expo Video, @gorhom/bottom-sheet, NativeWind (Tailwind), Ionicons, Supabase (auth, PostgREST, RPC, Realtime, Storage), FFmpeg Kit RN, Expo Notifications, Lottie RN (available), date-fns.
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)